EventHubs/tests/azure_metrics.py

```diff
@@ -0,0 +1,21 @@
+from azure.servicebus import ServiceBusService
+from requests import Session
+import json
+from datetime import datetime
+import time
```

Comment:
delete this file we already have test_eventhub_metrics.py create a function insert_mock_metrics_in_EventHub and in check error logs we can use this code https://learn.microsoft.com/en-us/python/api/overview/azure/monitor-query-readme?view=azure-python#resource-logs-query
you may also have to update the test dependencies in requirements.txt

Reply:
Deleted